make/debug

```diff
@@ -0,0 +1,7 @@
+DEBUG ?= 0
+ifeq ($(DEBUG), 1)
+	CXXFLAGS:=$(filter-out -O${O},$(CXXFLAGS))
+	GTEST_CXXFLAGS:=$(filter-out -O${O},$(GTEST_CXXFLAGS))
```

Comment:
also related to spacing, replace these tabs with two spaces, like the changes below. (I don't know off the top of my head if that's consistent with the rest of the makefile... go with what's more consistent.)